test/Microsoft.ML.OnnxTransformerTest/DnnImageFeaturizerTest.cs

```diff
@@ -89,28 +85,25 @@ void TestDnnImageFeaturizer()
         }
 
         [OnnxFact]
-        public void OnnxStatic()
+        public void OnnxFeaturizerWorkout()
         {
             var env = new MLContext(null);
             var imageHeight = 224;
             var imageWidth = 224;
             var dataFile = GetDataPath("images/images.tsv");
             var imageFolder = Path.GetDirectoryName(dataFile);
 
-            var data = TextLoaderStatic.CreateLoader(env, ctx => (
-                imagePath: ctx.LoadText(0),
-                name: ctx.LoadText(1)))
-                .Load(dataFile);
+            var data = ML.Data.LoadFromTextFile(dataFile, new[] {
+                new TextLoader.Column("imagePath", DataKind.String, 0),
+                new TextLoader.Column("name", DataKind.String, 1)
+            });
 
-            var pipe = data.MakeNewEstimator()
-                .Append(row => (
-                    row.name,
-                    data_0: row.imagePath.LoadAsImage(imageFolder).Resize(imageHeight, imageWidth).ExtractPixels(interleave: true)))
-                .Append(row => (row.name, output_1: row.data_0.DnnImageFeaturizer(m => m.ModelSelector.ResNet18(m.Environment, m.OutputColumn, m.InputColumn))));
+            var pipe = ML.Transforms.LoadImages("data_0", imageFolder, "imagePath")
+                .Append(ML.Transforms.ResizeImages("data_0", imageHeight, imageWidth))
+                .Append(ML.Transforms.ExtractPixels("data_0", interleavePixelColors: true))
+                .Append(ML.Transforms.DnnFeaturizeImage("output_1", m => m.ModelSelector.ResNet18(m.Environment, m.OutputColumn, m.InputColumn), "data_0"));
 
-            TestEstimatorCore(pipe.AsDynamic, data.AsDynamic);
```

Comment:
Do we still want to call `TestEstimatorCore` in this test?

Reply:
Yes thanks for pointing that out, I'll add it back.

---
In reply to: [304496249](https://github.com/dotnet/machinelearning/pull/4009#discussion_r304496249) [](ancestors = 304496249)